sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -74,7 +74,7 @@ any Bokeh model, using the ``js_on_change`` method of Bokeh models:
 
 It should be mentioned that the first parameter to ``js_on_change`` is
 actually the name of a BokehJS event. The full format for a property
-change event is, e.g. ``"change:start"`` but Bokeh will automatically
+change event is, e.g., "change:start", but Bokeh will automatically
```

Comment:
I think the backticks are appropriate to keep, this is a string literal that would be used/seen in code

Reply:
Absolutely, thanks for spotting this mistake. It'll be fixed in the next commit!